### PR TITLE
Single .env file

### PR DESCRIPTION
**Pull request by gravityblast**
Fixes: #3450 

## Checklist

This PR:

- [ ] Does it add new payout or donation token? In this case, have they been added to the indexer to avoid it to crash?
- [ ] Includes only changes relevant to the original ticket. Significant refactoring needs to be separated.
- [ ] Doesn't disable eslint rules.
- [ ] Doesn't work around the type checker (including but not limited to: type casts, non-null assertions, `@ts-ignore`, unjustified optional values).
- [ ] Doesn't contain commented out code.
- [ ] If adding/updating a feature, it adds/updates its test script on Notion.

### CODE REVIEW

File: .env.example
Comment (MasterHW) [inline .env.example:1]: would like to see this merged with https://github.com/gitcoinco/grants-stack/blob/streamline-env/.env.example

**Review by MasterHW — CHANGES_REQUESTED** [review]
i like the new IPFS/Pinata distinction. approved w/ one request

**Review by MasterHW — APPROVED** [review]
all good!

**Review by thelostone-mc — APPROVED** [review]
Do we need to make changes on env in production ?

**Pull request merged**